agoora-kafka-agent/src/main/java/io/spoud/agoora/agents/kafka/schema/confluent/ConfluentRegistrySchemaResource.java

```diff
@@ -13,6 +14,7 @@
 @RegisterRestClient(configKey = "rest-confluent-registry")
 @Path("/schemas")
 @RegisterClientHeaders(ConfluentAuthHeader.class)
+@RequestScoped
```

Comment:
This actually sounds wrong and more like the cause of a leak ;-)  should we put a comment here. We should put a comment here. I would just remove it if I see this and all tests pass afterwards.